package.json

```diff
@@ -110,7 +110,7 @@
     "@storybook/core-common": "next",
     "@storybook/csf": "next",
     "@storybook/csf-tools": "next",
-    "@storybook/store": "next",
+    "@storybook/preview-api": "next",
```

Comment:
I'd expect this change to have some runtime impact?

Reply:
fixed